proto/api.proto

```diff
@@ -725,6 +725,17 @@ service Api {
     };
   }
 
+  rpc PostubmitMineOre(PostSubmitMineOreRequest) returns (PostSubmitMineOreResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/mine-ore"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Submits two transactions in sequence for mining ore";
+      summary: "Submit mine ore transactions";
+    };
```

Comment:
please add a tag -  tags: ["Ore"];
this way it will be grouped separately in the swagger doc